modules/snooker/src/main/ts/ephox/snooker/api/Warehouse.ts

```diff
@@ -100,11 +109,19 @@ const justCells = function (warehouse: Warehouse) {
   return Arr.flatten(rows);
 };
 
+const justColumns = (warehouse: Warehouse) =>
```

Comment:
Name could maybe be clearer? Should it be "columns" or "groups"?

Reply:
Columns. You get the col-elements, not the colgroup.